src/lib/util.ts

```diff
@@ -28,12 +28,6 @@ export function copyToClipboard(value: any) {
   document.body.removeChild(el);
 }
 
-export function trace<T>(x: T, ...args: any[]): T {
-  // tslint:disable-next-line:no-console
-  console.debug("trace", ...args, x);
```

Comment:
I use trace sometimes!

Reply:
ok I'll restore that function in another PR